test/templates/helpers/templateHelpers.test.ts

```diff
@@ -2,3 +1,5 @@
+import {getMongoidPrimitive} from "../../../src/templates/helpers/templateHelpers";
```

Comment:
Codacy found an issue: ['getMongoosePrimitive' is defined but never used.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5738534)